spec/tictactoe_spec.rb

```diff
@@ -1,26 +1,117 @@
-require 'spec_helper' 
-require 'tictactoe' 
-describe TicTacToe do 
-  context 'when the board is viewed' do
-    it 'returns a single list with 9 elements' do 
-     game = TicTacToe.new
+require 'spec_helper'
+require 'tictactoe'
+## Faker Classes
+class FakeIO
+  attr_accessor :stdin, :stdout
+  def initialize(stdin: StringIO.new('') )
+      @stdin = stdin
+      @stdout = []
+  end
+
+  def puts( string )
+    @stdout << string
+  end
+
+  def gets
+    @stdin.gets @stdin.string
```

Comment:
If you wanted to make this a little simpler, you could use an array instead of a `StringIO`, where each element of the array resembles one user input.

```ruby
def gets
  @input.shift
end

FakeIO.new(input: ['4', '5']).gets # => 4
```

Reply:
Will do, that makes sense.